src/interface/src/styleguide/input/input-field.component.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChild,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { MatIconModule } from '@angular/material/icon';
+import { CommonModule } from '@angular/common';
+import { InputDirective } from './input.directive';
+
+export type ShowSupportMessage = 'always' | 'on-error' | false;
+
+/**
+ * Wrapper component for input fields.
+ * Expects an input[sgInput] element via content projection
+ *
+ * Usage:
+ * <sg-input-field>
+ *   <input sgInput ...>
+ * </sg-input-field>
+ */
+@Component({
+  selector: 'sg-input-field',
+  standalone: true,
+  imports: [MatIconModule, CommonModule, InputDirective],
+  templateUrl: './input-field.component.html',
+  styleUrl: './input-field.component.scss',
+})
+export class InputFieldComponent implements AfterContentInit {
+  /**
+   * Reference to the projected sgInput input element
+   */
+  @ContentChild(InputDirective, { static: false })
+  inputDirective?: InputDirective;
+
+  /**
+   * The help/error message to be displayed below the input field.
+   */
+  @Input() supportMessage = '';
+
+  /**
```

Comment:
Nit: should we clarify which icon library is currently valid for this?

Reply:
good point!